xep-0363.xml

```diff
@@ -32,6 +32,20 @@
     <email>daniel@gultsch.de</email>
     <jid>daniel@gultsch.de</jid>
   </author>
+  <revision>
+    <version>1.0.1</version>
+    <date>2021-12-28</date>
+    <initials>egp, mb</initials>
+    <remark>
+      <ul>
+        <li>Filename size in bytes.</li>
+        <li>Headers MUST be included in the PUT request.</li>
```

Comment:
This is not an editorial change -> needs a minor version bump (and will run by council).